### ISSUE TITLE
Empty Implementors section in doc

### ISSUE THREAD

**Issue by r00ster91** [rust-lang/rust#86356]
On https://doc.rust-lang.org/nightly/std/marker/trait.Sized.html#implementors the Implementors section is empty. I believe it should be omitted entirely, however above on that doc page it says "All type parameters have an implicit bound of `Sized`." so maybe it should say "All types implement this trait unless specified otherwise using `?Sized`." under that section? But that would kind of repeat the same information that we've already been given so maybe it's better to just omit it altogether.

See also: this issue's [Zulip topic](https://rust-lang.zulipchat.com/#narrow/stream/266220-rustdoc/topic/Empty.20implementors.20section).

**Comment by r00ster91** [comment]
@Stupremee I might misunderstand your claim but isn't this issue supposed to be discussed first to agree on what should be done before implementing something?

**Comment by Stupremee** [comment]
Yes it is. But I claimed it anyway so I can implement it once it's discussed

**Comment by crlf0710** [comment]
`Unsize` trait is also special.
https://doc.rust-lang.org/stable/std/marker/trait.Unsize.html#implementors

**Comment by lolbinarycat** [comment]
triage: yep, this is still an issue!
